### ISSUE TITLE
NPC Map Locations ignore redefined vanilla locations Toolips.

### ISSUE THREAD

**Issue by Yagisan** [Bouhm/stardew-valley-mods#73]
NPC Map Locations comes with all vanilla locations hardcoded.

To avoid a conflict with another mod, I moved the location and tooltip of the secret woods.

This moved location tooltip appears to ignored by NPC Map Locations, which continues to display the hardcoded tooltip in the previous location.

You can grab my mod which reproduces this, [from the Nexus here](https://www.nexusmods.com/stardewvalley/mods/8174).

Thank you for your assistance.

**Comment by bbpak** [comment]
Does the key used in `content.json` for the location name match the in-game location name exactly?

**Comment by Yagisan** [comment]
Yes, it does.  Here is a [direct link](https://github.com/Yagisan/Stardew_Valley/blob/5b65beb42d76a6bb2b31aa1e3ce203a49d98a809/Custom%20NPCs%20for%20NPC%20Map%20Locations/%5BCP%5D%20Yagisan%20-%20Custom%20NPCs%20for%20NPC%20Map%20Locations/content.json#L948) to the record on GitGub.

**Comment by Pathoschild** [comment]
This should be fixed in NPC Map Locations 3.0.0+, since it no longer overrides how the world map and its tooltips are drawn. Thanks for reporting it!
